Dockerfile.data

```diff
@@ -34,15 +34,15 @@ ENV LANGUAGE en_US.UTF-8
 # Upgrade pip
 RUN pip install --upgrade pip
 
+COPY requirements.txt /root/requirements.txt
+RUN pip --no-cache-dir install -r /root/requirements.txt
+
```

Comment:
File-comment - I think it's better to keep the postfixes matching the Docker image name, so `Dockerfile.nb`.